content/doc/book/pipeline/syntax.adoc

```diff
@@ -130,11 +138,16 @@ In `agents` declared within a stage, the options are invoked *before* entering t
 In this case, when using `timeout`, it is applied *before* the `agent` is allocated.
 
 ```
-timeout(unit: 'SECONDS', time: 5) {         // Options directive
-    stage("One"){                           // Stage level agents
-        node {
-            sleep 10
-            echo 'Hello'
+pipeline {                              
+    stages {
+        stage('Example') {
+            agent any                                         // Stage level agents
+            steps {
+                echo 'Hello World'
+                    options {                                 // Options directive
+                        timeout(time: 1, unit: 'SECONDS')   
+                    }
+            }
         }
```

Comment:
The options directive needs to precede the stage it is controlling, not be embedded with the steps
```suggestion
        stage('Example') {
            agent any                                         // Stage level agents
            options {
                timeout(time: 1, unit: 'SECONDS')   
            }
            steps {
                echo 'Hello World'
            }
        }
```

Reply:
May want to make it clear to the reader that the timeout embedded in the stage is only controlling timeout for exactly that stage.

Add another that clearly would exceed the timeout?

Reply:
> Add another that clearly would exceed the timeout?

By adding another do you mean adding another stage having timeout of different time period ?